src/bundle/Resources/public/js/OnlineEditor/core/base-richtext.js

```diff
@@ -151,8 +153,14 @@
 
         clearAnchor(element) {
             const icon = element.querySelector('.ez-icon--anchor');
-
-            if (icon) {
+            const elementPreviousSibling = element.previousSibling;
```

Comment:
`element.previousSibling` can be `null`, which would cause the script to crash due to method call on an undefined.

Reply:
The same comment as in https://github.com/ezsystems/ezplatform-admin-ui/pull/1745#discussion_r621045591.